src/main/java/core/gui/comp/table/FixedColumnsTable.java

```diff
@@ -1,11 +1,12 @@
 package core.gui.comp.table;
 
-import module.transfer.ui.sorter.DefaultTableSorter;
+import core.gui.comp.renderer.HODefaultTableCellRenderer;
+import core.model.HOConfigurationIntParameter;
 
 import javax.swing.*;
 import javax.swing.event.ListSelectionListener;
 import javax.swing.table.TableCellRenderer;
-import java.awt.*;
+import javax.swing.table.TableRowSorter;
 
 public class FixedColumnsTable extends JScrollPane {
```

Comment:
It is possibly because this is probably not the right inheritance here that we're having the issue above.  In all logic this should be a JTable which “decorates” (in the design pattern sense) a JTable in a scrollpane.  Maybe we'll want to refactor this in the future.

Reply:
extends JTable now
The surounding ScrollPane is returned in method getContainerComponent.

Reply:
It's not as easy as initially thought. There are some methods to override and some it doesn't work at all.